examples/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
do we need this new LICENSE compared to the LICENSE we already have in the root folder?

Reply:
Removed